frontend/src/Css.tsx

```diff
@@ -40,12 +40,14 @@ export const color = {
   successWeak: '#e6f4ea', // Google green 50
   terminated: '#80868b',
   theme: '#1a73e8',
-  themeDarker: '#0b59dc',
+  themeDarker: '#0049b5',
   warningBg: '#f9f9e1',
   warningText: '#ee8100',
   infoBg: '#f3f4ff',
   infoText: '#1a73e8',
   weak: '#9aa0a6',
+  link: '#0d47a1',
+  linkLight: '#5472d3',
```

Comment:
how did you get the two colors?
put them in a separate palette object first and then use them here?

Reply:
I select them from material UI recommended palette as shown below. May I ask what it means by `put them in a separate palette object`?

![palette](https://user-images.githubusercontent.com/37026441/125587964-c55c8ef7-b1dd-4c9e-8aee-b20cb77f371b.png)

Reply:
I meant sth like:
```js
const palette = {
  themeMain: 'xxxx',
  themeLight: 'aa',
  themeDark: 'cc',
}
const color = {
  linkLight: palette.themeMain,
  linkDark: palette.themeDark,
}
```

Reply:
Do we still need the color change after inventing the hover effect?

Reply:
Since ExecutionDetail page shows the link different from table view link, we are using the these fields that are newly defined.

I think the idea of `palette` is like defining the RGB code using color name, then reference them in `color` const with the name of utility/functionality. Does it make sense? Also, the current themeDark and linkDark actually have different meanings and usages, so we cannot use same color code for both of them.

Reply:
I found out that there is already a `palette` object in the same file. However, the usage is the reverse compared with the suggestion: `color` defines the RGB code, and `palette` references color for theme. 

We probably should rework our approach of defining css classes, but it is going to be out-of-scope of this PR. Following the existing patterns for now and we can update them together once we define the standard approach of creating palette.